govcd/vapptemplate.go

```diff
@@ -49,3 +49,20 @@ func (vdc *Vdc) InstantiateVAppTemplate(template *types.InstantiateVAppTemplateP
 	}
 	return nil
 }
+
+// Refresh refreshes the vApp template item information by href
+func (vAppTemplate *VAppTemplate) Refresh() error {
+
+	if vAppTemplate.VAppTemplate == nil {
+		return fmt.Errorf("cannot refresh, Object is empty")
+	}
+
+	url := vAppTemplate.VAppTemplate.HREF
```

Comment:
Shouldn't we error if the HREF is not set?

Reply:
fixed